projects/rocprofiler-systems/source/lib/core/trace_cache/metadata_registry.hpp

```diff
@@ -240,6 +242,10 @@ struct metadata_registry
     rocprofiler::sdk::callback_name_info_t<const char*> m_callback_tracing_info{
         rocprofiler::sdk::get_callback_tracing_names<const char*>()
     };
+    void overwrite_cb_names(
```

Comment:
```suggestion
    void overwrite_callback_names(
```
Consider renaming for clarity.